src/routes/+layout.svelte

```diff
@@ -1,9 +1,36 @@
 <script>
+  import { onMount } from 'svelte'
+
   import '../app.css'
   import Footer from '$lib/components/Footer.svelte'
   import Nav from '$lib/components/Nav.svelte'
+  import { slide } from 'svelte/transition'
+
+  let prevScrollPos = 0
+  let showNav = true
+
+  function handleScroll() {
+    const currentScrollPos = window.pageYOffset
+    showNav = prevScrollPos > currentScrollPos
+    prevScrollPos = currentScrollPos
+  }
+
+  onMount(() => {
+    document.title = 'HackHarvard 2023'
```

Comment:
Please do not change title dynamically here. Check [this for how to use the `svelte:head` component](https://svelte.dev/tutorial/svelte-head).